src/components/Notification/Notification.stories.mdx

```diff
@@ -26,15 +37,17 @@ Notifications are used to display global information, our notification variants
 
 <ArgsTable of={Notification} />
 
```

Comment:
I think it would be worth documenting somewhere on this page that we deprecated old props and introduced new ones.

So we could list which props were renamed: `type` -> `severity`, `close` -> `onDismiss`, `status` -> `title`.

Or maybe we could add this to Props table (to comments?):

<img width="1057" alt="Screenshot 2021-07-13 at 13 01 28" src="https://user-images.githubusercontent.com/83575/125441015-11afbc7a-3788-427b-bf4e-3345a750db39.png">

But I guess it may be easily missed when placed in there.